packages/utils/src/misc.ts

```diff
@@ -379,6 +379,11 @@ export async function handleDeeplinkRedirect({
     const env = getEnvironment();
 
     if (env === ENV_MAP.browser) {
+      if (!getDocument()?.hasFocus()) {
+        console.warn("Document does not have focus, skipping deeplink.");
```

Comment:
I think no log or just `console.log` would be more appropriate since this is the expected behaviour.